src/main/java/rocks/inspectit/gepard/agent/internal/instrumentation/InstrumentationState.java

```diff
@@ -50,10 +52,10 @@ public boolean isInstrumented(Class<?> clazz) {
    * @return true, if the provided type is already instrumented.
    */
   public boolean isInstrumented(InstrumentedType instrumentedType) {
```

Comment:
isIncluded() might be a better name, cause state.isInstrumented() sounds like asking if the state is instrumented.

Reply:
or exits()